nixos/modules/services/networking/ddclient.nix

```diff
@@ -52,6 +52,14 @@ in
         '';
       };
 
+      configFile = mkOption {
+        default = /etc/ddclient.conf;
```

Comment:
I think you ought to specify this as a string value.

Also, please add an example of using an alternative path, like `"/root/etc/secret/ddclient.conf"`.

Finally, I think it'd be good to explicitly state that the purpose of setting this option is to point to a file that won't be copied into the store.